unreleased.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+**Added:**
+- bpk-svgs
```

Comment:
Way more packages are affected...

- `bpk-component-icon`
- `react-native-bpk-component-icon`
- `react-native-bpk-component-button`
- (anything that exposes icons directly)


Check the changelog for previous addition to icons to see an example

Reply:
Thanks - now fixed!